packages/api/src/lib/mmt.ts

```diff
@@ -463,3 +463,28 @@ export const getSparqlQueryForLoRelationToNonDimConcept = (uri: string) => {
                               }`;
   return query;
 };
+
+export const getProblemObject = async (mmtUrl: string, concept: string) => {
```

Comment:
change variable name & console.error below. This is not a `concept` but `problemIdPrefix`